demo/admin/src/products/future/generated/ProductsGridInitialFitlerWithIsAnyOf.tsx

```diff
@@ -0,0 +1,199 @@
+import { gql, useApolloClient, useQuery } from "@apollo/client";
+import {
+    Button,
+    CrudContextMenu,
+    DataGridToolbar,
+    filterByFragment,
+    type GridColDef,
+    GridFilterButton,
+    muiGridFilterToGql,
+    muiGridSortToGql,
+    renderStaticSelectCell,
+    StackLink,
+    ToolbarActions,
+    ToolbarFillSpace,
+    ToolbarItem,
+    useBufferedRowCount,
+    useDataGridRemote,
+    usePersistentColumnState,
+} from "@comet/admin";
+import { Add as AddIcon, Edit as EditIcon } from "@comet/admin-icons";
+import { DamImageBlock } from "@comet/cms-admin";
+import { IconButton } from "@mui/material";
+import { DataGridPro, type GridSlotsComponent, GridToolbarQuickFilter } from "@mui/x-data-grid-pro";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import {
+    type GQLCreateProductMutation,
+    type GQLCreateProductMutationVariables,
+    type GQLDeleteProductMutation,
+    type GQLDeleteProductMutationVariables,
+    type GQLProductsGridInitialFilterWithIsAnyOfFragment,
+    type GQLProductsGridQuery,
+    type GQLProductsGridQueryVariables,
+} from "./ProductsGridInitialFitlerWithIsAnyOf.generated";
+
+const productsFragment = gql`
+    fragment ProductsGridInitialFilterWithIsAnyOf on Product {
+        id
+        title
+        status
+    }
+`;
+
+const productsQuery = gql`
+    query ProductsGrid($offset: Int!, $limit: Int!, $sort: [ProductSort!], $search: String, $filter: ProductFilter) {
+        products(offset: $offset, limit: $limit, sort: $sort, search: $search, filter: $filter) {
+            nodes {
+                ...ProductsGridInitialFilterWithIsAnyOf
+            }
+            totalCount
+        }
+    }
+    ${productsFragment}
+`;
+
+const deleteProductMutation = gql`
+    mutation DeleteProduct($id: ID!) {
+        deleteProduct(id: $id)
+    }
+`;
+
+const createProductMutation = gql`
+    mutation CreateProduct($input: ProductInput!) {
+        createProduct(input: $input) {
+            id
+        }
+    }
+`;
+
+function ProductsGridToolbar() {
+    return (
+        <DataGridToolbar>
+            <ToolbarItem>
+                <GridToolbarQuickFilter />
+            </ToolbarItem>
+            <ToolbarItem>
+                <GridFilterButton />
+            </ToolbarItem>
+            <ToolbarFillSpace />
+            <ToolbarActions>
+                <Button responsive startIcon={<AddIcon />} component={StackLink} pageName="add" payload="add">
+                    <FormattedMessage id="product.productsGridInitialFilterWithIsAnyOf.newEntry" defaultMessage="New Product" />
+                </Button>
+            </ToolbarActions>
+        </DataGridToolbar>
+    );
+}
+
+export function ProductsGrid() {
+    const client = useApolloClient();
+    const intl = useIntl();
+    const dataGridProps = {
+        ...useDataGridRemote({
+            initialFilter: {
+                items: [{ field: "status", operator: "isAnyOf", value: "Published,Unpublished" }],
```

Comment:
I guess that's the problem: This probably shouldn't be a string, but an array.

Reply:
Exactly, thats the problem of the generated code. 👍